localnet/single/start.sh

```diff
@@ -4,11 +4,11 @@ command -v shellcheck >/dev/null && shellcheck "$0"
 
 # Please keep this in sync with the Ports overview in HACKING.md
 TENDERMINT_PORT_GUEST="26657"
-TENDERMINT_PORT_HOST="26457"
+TENDERMINT_PORT_HOST="26657"
```

Comment:
This part was set like this so that it does not be conflicted with with-ibc localnet. If it change it like this, it will be conflicted, is that ok?

Reply:
Yes, I think it is no problem. Because if someone use `with-ibc` docker-compose, I think he/she will turn off a single localnet docker-compose.